spec/support/shared_examples_policy_machine.rb

```diff
@@ -220,6 +220,25 @@
     end
   end
 
+  describe 'Operations' do
+
+    it 'does not allow an operation to start with a ~' do
+      expect{policy_machine.create_operation('~_~')}.to raise_error(ArgumentError)
+      expect{policy_machine.create_operation('apple~')}.not_to raise_error
+    end
+
+    it 'can derive a prohibition from an operation and vice versa' do
+      @op = policy_machine.create_operation('fly')
+      expect(@op.prohibition).to be_prohibition
+      expect(@op.prohibition.operation).to eq('fly')
+    end
+
+    it 'raises if trying to negate a non-operation' do
```

Comment:
There should be a test for negating a legitimate operation <code>PM::Prohibition.new('apple')</code>